overlord/snapstate/snapstate_update_test.go

```diff
@@ -1035,6 +1043,8 @@ func (s *snapmgrTestSuite) TestUpdateRunThrough(c *C) {
 	err = snapstate.Get(s.state, "services-snap", &snapst)
 	c.Assert(err, IsNil)
 
+	c.Assert(snapst.LastRefreshTime, NotNil)
+	c.Check(*snapst.LastRefreshTime, Equals, now)
```

Comment:
the other check did `c.Check(*snapst.LastRefreshTime.Equals(now), Equals, true)` should we do the same here?

Reply:
This however looks much more readable, unless there's some subtle difference I'm not aware of, I'd vote for using this style of checking.

Reply:
The way @anonymouse64 suggested is correct, while using just Equals (or DeepEquals) on time struct sometimes gives weird differences depending on how time.Time was constructed, probably due to subtle differences of some time fields.